packages/syft/src/syft/core/adp/vectorized_publish.py

```diff
@@ -155,7 +157,20 @@ def publish(
         # Step 2: Calculate the epsilon spend for this query
 
```

Comment:
Q: Would it be possible to catch these(the nan value errors) on the DS side and intimate them(ex: based on the min, max values, and the computation performed), or are we planning for client-side checks in `0.8`?

Reply:
Great idea Rasswanth, added in [3e18454](https://github.com/OpenMined/PySyft/pull/7008/commits/3e184546b926986b6a293cf90beb8cb1d541d050)